tests/test_oclc.py

```diff
@@ -245,37 +268,43 @@ def test_all_authors_get_viaf_lookup(self):
         # a sort name + VIAF, and one with a VIAF but no sort name.
         contributor1 = ContributorData(viaf="1")
         contributor2 = ContributorData(viaf="2", sort_name="Jordan, Robert")
+        contributor3 = ContributorData(sort_name="Rice, Anne", display_name="Anne Rice")
         idata = IdentifierData(type=identifier.type, 
                                identifier=identifier.identifier)
         metadata = Metadata(
             DataSource.OCLC_LINKED_DATA,
-            contributors=[contributor1, contributor2],
+            contributors=[contributor1, contributor2, contributor3],
             primary_identifier=idata,
             title=u"foo"
         )
         oclc.queue_info_for(metadata)
 
         # Our OCLC Linked Data client is going to try to fill in the
-        # data, asking VIAF about the contributors.
+        # data, asking VIAF about the contributors that have VIAF data,
+        # and not those who do not.
         lookup1 = (ContributorData(
                   viaf="1", display_name="Display Name",
                   family_name="Family", sort_name="Name, Sort",
                   wikipedia_name="Wikipedia_Name"), None, None)
         lookup2 = (ContributorData(
                    viaf="2", wikipedia_name="Robert_Jordan_(Author)",
                    biography="That guy."), None, None)
-        viaf.queue_lookup(lookup1, lookup2)
+        viaf.queue_lookup(lookup1, lookup2, "Unrequested lookup")
```

Comment:
I verify a few lines below (unless you're saying it's not necessary to do so?). In the latest commit I moved that verification and added a bit more of a comment to clarify.